vertx-web/src/main/java/io/vertx/ext/web/handler/CSRFValidationException.java

```diff
@@ -0,0 +1,12 @@
+package io.vertx.ext.web.handler;
```

Comment:
I'd rather see these exception type inside the `impl` package as we may further change it to a hierarchy.

Also as there are no tests, I'm not sure how this will be used differntly than:

```java
  @Test
  public void testCatchingCustomExceptions() throws Exception {
    String path = "/cors";
    router.route(path)
      .handler(ctx -> {
        ctx.fail(403, new IllegalStateException("validation error!"));
      })
      .failureHandler(ctx -> {
        if (ctx.failure() instanceof IllegalStateException){
          // OK
          ctx.end();
        } else {
          fail("Not expected!");
        }
      });

    testRequest(
      HttpMethod.GET,
      "/cors",
      200, "OK");
  }
```

Reply:
Thanks for the review. I will adjust it